tests/utils/concat-tests-test.js

```diff
@@ -16,8 +16,9 @@ var fixtureTestPath = path.join(__dirname, 'fixtures/packages/ember-metal/tests'
 
 describe('concat-es6', function() {
```

Comment:
@rwjblue this refactor maintains the original behavior, but this test is failing:

```
  1) concat-es6 correctly concats test tree into one file properly:
     Error: Path or pattern "**/*.js" did not match any files
    at Object.multiGlob (/Users/Sean/src/emberjs-build/node_modules/broccoli-static-compiler/node_modules/broccoli-kitchen-sink-helpers/index.js:202:13)
    at /Users/Sean/src/emberjs-build/node_modules/broccoli-static-compiler/index.js:25:27
    at $$$internal$$tryCatch (/Users/Sean/src/emberjs-build/node_modules/broccoli/node_modules/rsvp/dist/rsvp.js:490:16)
    at $$$internal$$invokeCallback (/Users/Sean/src/emberjs-build/node_modules/broccoli/node_modules/rsvp/dist/rsvp.js:502:17)
    at $$$internal$$publish (/Users/Sean/src/emberjs-build/node_modules/broccoli/node_modules/rsvp/dist/rsvp.js:473:11)
    at Object.$$rsvp$asap$$flush [as _onImmediate] (/Users/Sean/src/emberjs-build/node_modules/broccoli/node_modules/rsvp/dist/rsvp.js:1581:9)
    at processImmediate [as _immediateCallback] (timers.js:354:15)
```

It is unclear how I should properly stub this out to play nice with the packages in `tests/fixtures`.

Any ideas?